src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/client/transformation/TransformationClientTest.java

```diff
@@ -145,10 +145,10 @@ public void should_throw_case_transformation_exception_when_unable_to_process_bo
         );
 
         // then
-        assertThat(throwable).isInstanceOf(CaseTransformationException.class);
+        assertThat(throwable).isInstanceOf(CaseClientServiceException.class);
 
         // and
-        CaseTransformationException exception = (CaseTransformationException) throwable;
+        CaseClientServiceException exception = (CaseClientServiceException) throwable;
```

Comment:
On the same go we could improve this by `catchThorableOfInstance` or what this method is called. This way we could move out these 2 sections and make `then` section applicable from first assertion above following lines

P.S. same recommendation applied in entire PR

Reply:
Done